CoreImportExportPlugins/test/unit/src/au/gov/asd/tac/constellation/plugins/importexport/ConfigurationPaneNGTest.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2010-2022 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.importexport;
+
+import au.gov.asd.tac.constellation.graph.Attribute;
+import static au.gov.asd.tac.constellation.graph.attribute.interaction.AttributeValueTranslator.LOGGER;
```

Comment:
Instead of importing another logger (which should be private, not sure why that one isn't), are you able to create one specific for this class